import_activity_events.py

```diff
@@ -59,7 +65,7 @@
 """
 
 Q_CREATE_EVENTS_TABLE = """
-    CREATE TABLE IF NOT EXISTS activity_events (
+    CREATE TABLE IF NOT EXISTS activity_events_{sample_rate} (
```

Comment:
This assumes that the existing queries are only interested in the last 3 months of data (or whatever number we decide on). Is that the case?

In that sense, I kind of saw the enforced name change as a feature. Query authors should make a conscious decision whether they prefer breadth or depth.

(although also, I'm happy to change it if you think that's senseless tyranny)